### ISSUE TITLE
Study/Trial contacts

### ISSUE THREAD

**Issue by feserm** [plantbreeding/API#582]
I have some questions regarding the `contacts` field in Study and Trial:
- Is there a reason for why this is not using the Person Entity but using its own scheme?
- If not using Person, is the Contact created through `POST /studies` and `POST /trials`? And why is the `contactDbId` set by the user? Or how would someone ensure, that it is uniquely identifying?

**Comment by BrapiCoordinatorSelby** [comment]
I agree the Person entity and Contact entity should be more aligned.

I believe that creating a Contact through `POST /studies` and `POST /trials` is still important because there are use cases where the contact person might not be a registered user in the system, or a particular database doesn't support the `/people` calls for accessing their users.

**Comment by sebastian-raubach** [comment]
Just to add another opinion to this, I'm just looking at various endpoints that reference some kind of individual and it seems this would be a candidate for some unification across the whole API. For example, there is the `/people` endpoint which can be associated with a `program` or a `listdetail`. At the same time, `trial` has a contacts field that (from what I understand) is aligned with a MIAPPE person. Then there's also the quite generic `uploadedBy` (string) on POST `/observations` which says it can be either a name or an id. I feel like all these use-cases would benefit from reusing the same people entity that already exists within BrAPI.

So, basically, I agree with @feserm but would potentially also include the `uploadedBy` field in `observation` into the same entity.
